db/vita_partners.yml

```diff
@@ -464,11 +464,18 @@ vita_partners:
   display_name: United Way of Racine County
   source_parameters:
   - unitedwayracine
-  logo_path: ''
+  logo_path: 'VITA & UWRC Logo - Horizontal RGB Lockup.png'
```

Comment:
This needs to start with `partner-logos/`. Also - it would be best to rename these files to not contain any special characters or spaces.

Reply:
No problem Tom will do.